cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/flow/FlowletProgramRunner.java

```diff
@@ -53,8 +53,8 @@
 import co.cask.cdap.common.metrics.MetricsCollectionService;
 import co.cask.cdap.common.metrics.MetricsCollector;
 import co.cask.cdap.common.queue.QueueName;
-import co.cask.cdap.data.stream.StreamCoordinator;
 import co.cask.cdap.data.stream.StreamPropertyListener;
+import co.cask.cdap.data.stream.service.StreamCoordinator;
```

Comment:
Shouldn't move the Coordinator. The `StreamCoordinator` is part of stream system, not just for stream service.